tcms/testplans/static/testplans/js/get.js

```diff
@@ -162,6 +161,20 @@ function attachEvents(testPlanId, permissions) {
         });
     }
 
+    // get details and draw expand area only on expand
+    $('.js-testcase-row').click(function(ev) {
+        const testCaseId = getCaseIdFromEvent(ev);
+
+        // tc was expanded once, dom is ready
+        if (expandedTestCaseIds.indexOf(testCaseId) > -1) {
+            return;
+        }
+
+        const tcRow = $(ev.target).closest('.list-group-item');
```

Comment:
Isn't this the closest `.js-testcase-row` ? you also have the data attribute so you can query & match by the specific ID